applications/settings/desktop_settings/scenes/desktop_settings_scene_favorite.c

```diff
@@ -212,21 +212,25 @@ bool desktop_settings_scene_favorite_on_event(void* context, SceneManagerEvent e
                 strncpy(
                     curr_favorite_app->name_or_path,
                     furi_string_get_cstr(temp_path),
-                    MAX_APP_LENGTH);
+                    sizeof(curr_favorite_app->name_or_path));
```

Comment:
why?

Reply:
MAX_APP_LENGTH might be some weird number (for example, `name_or_path` might be defined as `char name_or_path[MAX_APP_LENGTH + 1]`), better rely on the type information.